### DISCUSSION TITLE
Dissociate a music file from a playlist

### DISCUSSION THREAD

**Discussion by nathanfriend** [AzuraCast/AzuraCast#3804] [Q&A]
How do you dissociate a music file from a playlist without deleting or setting to anther playlist?

**Comment by BusterNeece** [comment]
Select it and either set it to no playlists (just click "Set" with nothing checked) or click "Clear" in the playlists dropdown.

**Reply by nathanfriend** [reply]
Oh wow, that's simple not sure why I didn't try that.  Thanks.
